contracts/test/CompoundGovernorInitialize.t.sol

```diff
@@ -0,0 +1,17 @@
+// SPDX-License-Identifier: BSD-3-Clause
```

Comment:
Is it a good idea that we're breaking this and set quorum tests out of `CompountGovernor.t.sol` and to their own separate contracts? And are we planning on doing the same for all the rest of the functions we test?

Reply:
My thinking here was that we could isolate each unique function to the new governor to smaller unique test files, hopefully making the review process quicker. If the use of individual files seems burdensome, can certainly switch back.

Reply:
i like your thinking there @jferas. i love a tight review process! hmm, I'm curious how it would go if we broke each method into a new file. Would there be too many files, i.e. hard to manage? or would it be way better than a monolithic CompoundGovernor.t.sol? I'm impartial here, an experiment could be interesting, what do you think @garyghayrat @jferas ?

Reply:
I was thinking that we'd only be unit testing new/unique features of the governor, so the number of files wouldn't be too large. Also, as each of us would likely be working on a single new feature at a time, this would minimize that location where we're touching the same file to the governor itself.. the test files would be unique to the feature and we'd have fewer possibilities of rebase conflicts that way.

Reply:
I'm open to experimenting!

Reply:
@jferas @garyghayrat just calling out that there might be some initial merge conflicts on your two PRs as a result of this